services/src/datasets/external/netcdfcf/error.rs

```diff
@@ -169,4 +169,14 @@ pub enum NetCdfCf4DProviderError {
     CannotGenerateLoadingInfo {
         source: Box<dyn ErrorSource>,
     },
+    CannotCreateInProgressFlag {
+        source: Box<dyn ErrorSource>,
+    },
+    CannotRemoveInProgressFlag {
+        source: Box<dyn ErrorSource>,
+    },
+    CannotRemoveOverviewsWhileCreationIsInProgress,
```

Comment:
one could argue that we should abort the creation process in that case but I guess it's ok to return an error

Reply:
Yeah, so maybe you want that or maybe not… I would leave it as is if it is okay for you.